templates/platforms/android-studio/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Tue May 10 19:22:52 CST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

Comment:
are you sure this works?	 tauri is on 8.0 btw

Reply:
yeah works fine on my Windows machine, minor gradlew versions are usually safe to upgrade to.